### PR TITLE
SRCH-3046 update the i14y README to use search-services

### PR DESCRIPTION
**Pull request by danswick**
## Summary
- Updates to README:
  * Describe new `search-services` method for starting relevant services.
  * Move service-specific instructions to the search-services README:
    * [x] TODO: Create PR for search-services docs and link here: https://github.com/GSA/search-services/pull/5
- Delete docker/docker compose files.
- Made minor version bumps in circle config.
- Minor env variable tweak to match other repos: `ES_HOST` --> `ES_HOSTS`
 
### Checklist
Please ensure you have addressed all concerns below before marking a PR "ready for review" or before requesting a re-review:
 
#### Functionality Checks
 
- [x] Code is functional.

- [x] Automated checks pass. If CodeClimate checks do not pass, explain reason for failures:

- [x] If your changes will be tested manually, you have run `bundle update` and committed your changes to Gemfile.lock.
 
- [x] You have merged the latest changes from the target branch (usually `main`) into your branch.
 
- [x] Your target branch is `main` or `production`, or you have specified the reason for an alternate branch here:
 
- [x] PR title is either of the format **SRCH-#### \<description\>** matching the associated Jira ticket (i.e. "SRCH-123 implement feature X"), or **Release - SRCH-####, SRCH-####, SRCH-####** matching the Jira ticket numbers in the release.
 
- [x] You have squashed your commits into a single commit (exceptions: your PR includes commits with formatting-only changes, such as required by Rubocop or Cookstyle, or if this is a feature branch that includes multiple commits). **N/A**: I plan to use "squash and merge" instead of a manual merge.
 
-  [x] Your primary commit message is of the format **SRCH-#### \<description\>** matching the associated Jira ticket. **N/A**: by using "squash and merge", the title of this PR will become the commit message.
 
#### Process Checks

- [x] You have specified an "Assignee", and if necessary, additional reviewers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,19 +13,15 @@ Search engine for agencies' published content
...
-    * [analysis-smartcn](https://www.elastic.co/guide/en/elasticsearch/plugins/current/analysis-smartcn.html)
+### Docker Services
 
-We recommend using [Docker](https://www.docker.com/get-started) to install and run Elasticsearch:
+The required services (MySQL, Elasticsearch, etc.) can be run using Docker. Please refer to [searchgov-services](https://github.com/GSA/search-services) for detailed instructions on centralized configuration for services.
 
-```
-$ docker-compose up elasticsearch
-```
+The Elasticsearch services provided by `searchgov-services` is configured to run on the default port, [9200](http://localhost:9200/). To use a different host (with or without port) or set of hosts, set the `ES_HOSTS` environment variable. For example, use following command to run the specs using Elasticsearch running on `localhost:9207`:
```
Comment (danswick) [inline README.md:20]: When would someone want to override the default port? Are there scenarios other than testing with Elasticsearch 7 running on `9207`?

File: config/elasticsearch.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 default: &DEFAULT
   hosts:
-    - <%= ENV['ES_HOST'] || 'localhost:9200' %>
+    - <%= ENV['ES_HOSTS'] || 'localhost:9200' %>
```
Comment (danswick) [inline config/elasticsearch.yml:3]: Made this plural to match [`search-gov`](https://github.com/GSA/search-gov/blob/e285f8a54b44694cefd781f01c0be084726dec98/config/secrets.yml.dev#L61).

**Comment by MothOnMars** [comment]
@danswick , can you complete the remaining checklist items? If an item doesn't apply or is not nececessary (such as merging recent changes from `main`, when there aren't any), you can check the box and add an "N/A" to the checklist item.

**Comment by danswick** [comment]
@MothOnMars PR checklist has been updated!

File: README.md [hunk 3]
```diff
@@ -13,19 +13,15 @@ Search engine for agencies' published content
...
 
-Verify that Elasticsearch 6.8.x is running on port 9200:
+    ES_HOSTS=localhost:9207 bin/rspec spec
+
+Verify that Elasticsearch 6.8.x is running on port the expected port (in this example, port 9200):
```
Comment (lorawoodford) [inline README.md:24]:
```suggestion
Verify that Elasticsearch 6.8.x is running on the expected port (port 9200 by default):
```

**Review by lorawoodford — CHANGES_REQUESTED** [review]
@danswick Same small typo as search-gov.

**Review by lorawoodford — APPROVED** [review]
LGTM

**Pull request merged**